step1-build-base-fonts.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+# Creates subsetted versions of the original fonts that will serve as the base for producing IFT fonts.
+# Notably this splits the CJK families each into two distinct subsets.
```

Comment:
I'd like to ask for advice, why does the CJK font need to be split into two files here?

Reply:
This is a size optimization I was experimenting with. At a high level: when using [glyph keyed](https://w3c.github.io/IFT/Overview.html#glyph-keyed) patches only glyph outline data is augmented and data for the supporting tables (for example hmtx) must already be in the font. So the full font is split into two pieces a high frequency one and a low frequency one. Then if you don't need any of the low frequency characters you avoid loading all of the auxillary data for them.

However, since I set this up the spec and our understanding of encoding approaches has advanced and we're able to get a similar effect by mixing [table keyed](https://w3c.github.io/IFT/Overview.html#table-keyed) patches with glyph keyed, so I'm not sure this optimization is needed anymore. Might be worthwhile to try and update the approach used here and see how that fairs.